orchestrator/gorc/Cargo.toml

```diff
@@ -40,13 +40,11 @@ prost = "0.7"
 bytes = "1"
 
 [dependencies.abscissa_core]
-git =  "https://github.com/iqlusioninc/abscissa"
-rev = "dccb38ad66afed9506e152733deec6ef40d17f53"
-#version = "0.6.0-pre.1"
+version = "0.6.0-pre.1"
 # optional: use `gimli` to capture backtraces
 # see https://github.com/rust-lang/backtrace-rs/issues/189
 # features = ["gimli-backtrace"]
 
 [dev-dependencies]
-abscissa_core = { git = "https://github.com/iqlusioninc/abscissa", rev = "dccb38ad66afed9506e152733deec6ef40d17f53" } #{ version = "0.6.0-pre.1", features = ["testing"] }
+abscissa_core = { version = "0.6.0-pre.1", features = ["testing"] }
```

Comment:
should the version of dev-dependencies and main ones match? I remember I had to change dev-dependencies, because it didn't build otherwise

Reply:
The abscissa_core only has version `0.6.0-pre.1` released, no `0.6.0-pre.2` for it.